### PR TITLE
BOM update - rules-base;  WINDUP-486  GraphService#reload( WindupVertexM...

### PR DESCRIPTION
**Pull request by OndraZizka**
...odel )

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Merged build triggered.

File: graph/api/src/main/java/org/jboss/windup/graph/service/GraphService.java [hunk 1]
```diff
@@ -298,14 +299,20 @@ public TitanTransaction newTransaction()
...
-    public static <T extends WindupVertexFrame> T addTypeToModel(GraphContext graphContext, WindupVertexFrame frame,
-                Class<T> type)
+    public static <T extends WindupVertexFrame> T addTypeToModel(GraphContext graphContext, WindupVertexFrame frame, Class<T> type)
     {
         Vertex vertex = frame.asVertex();
         graphContext.getGraphTypeRegistry().addTypeToElement(type, vertex);
         return graphContext.getFramed().frame(vertex, type);
     }
 
+    public T reload(T frame)
```
Comment (jsight) [inline graph/api/src/main/java/org/jboss/windup/graph/service/GraphService.java]: This is missing javadoc, and I'm not sure that I like the name. "reload" implies that it is reloading the Vertex, but it is actually just reframing the already loaded vertex. Thoughts?

File: bom/pom.xml [hunk 2]
```diff
@@ -162,8 +162,18 @@
             </dependency>
 
 
+
+            <!-- Windup Rules Base -->
+            <dependency>
```
Comment (jsight) [inline bom/pom.xml]: Great!

**Comment by windupgithubbot1** [comment]
Merged build triggered.

**Comment by windupgithubbot1** [comment]
Merged build started.

**Comment by windupgithubbot1** [comment]
Merged build finished. Test FAILed.

**Comment by windupgithubbot1** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://10.3.9.192//job/windup-pr-builder/406/
Test FAILed.

File: graph/api/src/main/java/org/jboss/windup/graph/service/GraphService.java [hunk 3]
```diff
@@ -22,6 +22,7 @@
 import com.tinkerpop.frames.VertexFrame;
 import com.tinkerpop.frames.modules.typedgraph.TypeValue;
 import com.tinkerpop.gremlin.java.GremlinPipeline;
+import org.jboss.windup.qs.identarch.model.IdentifiedArchiveModel;
```
Comment (jsight) [inline graph/api/src/main/java/org/jboss/windup/graph/service/GraphService.java]: This line does not compile.

**Comment by lincolnthree** [comment]
Closing this PR. Please include this in the PR for your larger GAV feature.

File: bom/pom.xml (hunk 2 above)
Comment (OndraZizka) [inline bom/pom.xml]: Since this PR was rejected, I replicated this in https://github.com/windup/windup/pull/428

**Pull request closed without merging**